src/c2s/mongoose_c2s.erl

```diff
@@ -217,6 +217,7 @@ activate_socket(#c2s_data{socket = Socket}) ->
 
 -spec send_text(c2s_data(), iodata()) -> ok | {error, term()}.
 send_text(#c2s_data{socket = Socket}, Text) ->
+    mongoose_metrics:update(global, [data, xmpp, sent, xml_stanza_size], size(Text)),
```

Comment:
Careful, because `size/1` will only take a tuple or a binary, but `exml:to_iolist/1` might return here an iolist of binaries if a list of `#xmlel{}` records was given. Ideally, this should use `iolist_size/1` instead. Will also (theoretically) give better type information.